.github/workflows/create-release.yml

```diff
@@ -38,18 +38,28 @@ jobs:
       - name: Edit pyproject.toml
         run:  sed -i 's/^\(version *= *\).*$/\1"${{ inputs.version }}"/' pyproject.toml
 
-      - name: Configure git
-        run:  |
-          git config --global user.name 'BotReleaser'
-          git config --global user.email 'bot.releaser@users.noreply.github.com'
-
-      - name: Adding file
-        run:  |
-          git add pyproject.toml
-          git commit -m "v${{ inputs.version }}" --allow-empty
-          git tag v${{ inputs.version }}
-
-      - name: Push to main and tags
-        run: |
-          git push origin main
-          git push origin v${{ inputs.version }}
+      - name: Setup PDM
+        uses: pdm-project/setup-pdm@v3
+        with:
+          python-version: '3.10'
+          cache: false
+
+      - name: Build release candidate wheel
+        run: pdm build
+
+      - name: Upload RC wheel artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: rc-wheel
```

Comment:
why not rc-wheel-${version} ?

Reply:
Yes let's go for that, I'm all for more precise names 👍